packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -571,6 +571,30 @@ describe('e2e', function () {
       shell.assertNoErrors();
     });
 
+    it('prints only multiline dots between prompts', async function () {
+      await shell.waitForPrompt();
+      const outputOffsetBefore = shell.output.length;
+      shell.writeInputLine(`
+        const foo = {
+          bar: "baz"
+        };
+      `);
+
+      // Wait for some multiline dots
+      await eventually(() => {
+        shell.assertContainsOutput('... ...');
+      });
+      const outputOffsetAfterDots = shell.output.length;
+      const output = shell.output
+        .substring(outputOffsetBefore, outputOffsetAfterDots)
+        .trim();
+      // Expect just the multiline dots
+      expect(output.startsWith('... ...')).equals(
```

Comment:
This test does fail for me, locally, but unfortunately I think that's actually expected behavior? The first prompt printed by mongosh in response to the `.writeInputLine()` call is a regular prompt, not a multiline prompt, but that's happening because the first line of the `writeInputLine()` argument is empty/whitespace-only.

That being said ... yeah, there's a good chance that that causes the flakiness in the FLE test at least. I feel like it's probably a good idea to just always `.trim()` the input to `.writeInputLine()`? For raw input, there's always `.writeInput()` itself